### PR TITLE
Feature/home tabbar ui

### PR DESCRIPTION
**Pull request by pnhanhbk**
#6 
#7 
#8

### CODE REVIEW

**Comment by mrvandai** [comment]
:+1: There are no linter violations.

File: Meete/Define/Colors.swift [hunk 1]
```diff
@@ -12,4 +12,7 @@ import SwiftUtils
 final class Color {
     static let white = UIColor.white
```
Comment (mrvandai) [inline Meete/Define/Colors.swift]: Please remove this, DO NOT define system color.
please use UIColor.white

File: Meete/Controllers/CategoryCell.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import UIKit
+
+class CategoryCell: UICollectionViewCell {
+
+    @IBOutlet private weak var cafeIcreamButton: UIButton!
+    @IBOutlet private weak var nccButton: UIButton!
+    @IBOutlet private weak var eatingHouseButton: UIButton!
+    @IBOutlet private weak var restaurantButton: UIButton!
+    @IBAction func selectCategory(_ sender: Any) {
```
Comment (mrvandai) [inline Meete/Controllers/CategoryCell.swift]: Why action at this line
Please using // MAKR: Actions

File: Meete/Controllers/CategoryCell.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
```
Comment (mrvandai) [inline Meete/Controllers/CategoryCell.swift]: Move selectCategory action below here, remember add // MARK: Actions

File: Meete/Controllers/HomeViewController.swift [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+//
+//  Created by Pham Ngoc Hanh on 4/27/17.
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: BaseViewController {
+
+    @IBOutlet private weak var homeCollectionView: UICollectionView!
```
Comment (mrvandai) [inline Meete/Controllers/HomeViewController.swift]: Please using tableView

File: Meete/Define/Colors.swift (hunk 1 above)
Comment (pnhanhbk) [inline Meete/Define/Colors.swift]: OK. It is example  available in the project template

**Pull request merged**